modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,212 @@
+                        ""system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster""
+                    ]
+                }
+            }
+        }
+    ]
+}"
+----
+<1> Replace '{AWS_AccountNo}' with your AWS account number.
+<2> Replace '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in
+step 1b.
+
+[NOTE]
+====
+Do not include the 'https' portion of the OIDC DNS url when replacing '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in
+step 1b.  Only the alphanumeric information that proceeds the '.net/' within the URL is needed.
+====
+--
+. Add the tags necessary for subnet discovery:
+
+.. Add the following {Key: Value} tag to the VPC hosting the {product-title}  cluster, replacing '{Cluster Infra ID}' with the Infra ID specified in step 1a:
+
+
+* kubernetes.io/cluster/${Cluster Infra ID}:owned
+
+.. Add the following ELB V2 {Key: Value} tags to the private subnets and (optional) public subnets:
+
+* Private subnets: kubernetes.io/role/internal-elb:1
+* Public subnets: kubernetes.io/role/elb:1
++
+[NOTE]
+====
+Internet-facing and internal load balancers will be created within the AZ of which these subnets belong.
+====
++
+For more information on adding tags to AWS resources (including VPCs and subnets), see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tag your Amazon EC2 resources].
++
+. Log into the {product-title} cluster as the Cluster-Admin and create a new project using the following command:
++
+[source, terminal]
+----
+$ oc new-project aws-load-balancer-operator
+----
++
+. Link the AWS IAM role created in step 2a with the AWS ALBO service account and create a Kubernetes Secret for the AWS ALBO:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+stringData:
+  credentials: |
+    [default]
+    role_arn = <aws-load-balancer-operator-role-arn><1>
+    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+----
+<1> Replace <aws-load-balancer-operator-role-arn>' with the AWS IAM role created in step 2a.
++
+. Create AWS ALBO:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  upgradeStrategy: Default
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1.0
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: aws-load-balancer-operator.v1.0.0
+----
++
+. Create AWS ALBC:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1
+kind: AWSLoadBalancerController
+metadata:
+  name: cluster
+spec:
+  credentials:
+    name: aws-load-balancer-operator
+----
++
+[IMPORTANT]
```

Comment:
Covers row 47 in https://docs.google.com/spreadsheets/d/1nwUdersMAaKZ4AUX_KhCpgsjL6DByxVy2B95CNqaeYY/edit#gid=1241421272